packages/@ourworldindata/grapher/src/entitySelector/EntitySelector.tsx

```diff
@@ -947,48 +981,35 @@ export class EntitySelector extends React.Component<{
     }
 
     private renderFooter(): JSX.Element {
-        const { numSelectedEntities, selectedEntityNames } = this.selectionArray
+        const { numSelectedEntities } = this.selectionArray
         const { partitionedVisibleEntities: visibleEntities } = this
 
         return (
             <div className="entity-selector__footer">
-                {this.isMultiMode ? (
-                    <>
-                        <div className="footer__selected">
-                            {numSelectedEntities > 0
-                                ? `${numSelectedEntities} selected`
-                                : "Empty selection"}
-                        </div>
-                        <button
-                            type="button"
-                            onClick={this.onClear}
-                            disabled={visibleEntities.selected.length === 0}
-                        >
-                            Clear
-                        </button>
-                    </>
-                ) : (
-                    <div className="footer__selected">
-                        {selectedEntityNames.length > 0 ? (
-                            <>
-                                Current selection:
-                                <span className="entity-name">
-                                    {selectedEntityNames[0]}
-                                </span>
-                            </>
-                        ) : (
-                            "Empty selection"
-                        )}
-                    </div>
```

Comment:
Was removing this also due to Marwa's feedback? I felt like it worked quite well.